.script/utils/LogoChecker/logoImageSVGChecker.ts

```diff
@@ -0,0 +1,97 @@
+import { LogoValidationError } from "../validationError";
+
+export function isValidLogoImageSVGContent(LogoImagesContent: string) {
+  if (isLogoSVGHasStyleTag(LogoImagesContent)) {
```

Comment:
nit: for conciseness, consider using a generic function and iterate illegal attributes
```
illegalAttribute = ["style=", "cls=",....]
illegalAttibutes.forEach(illegalAttribute => {
  if LogoImagesContent.includes(illegalAttribute) {
        throw new LogoValidationError(`Raw file of logo should not contain ${illegalAttribute}`);
  }
}
```